src/hoverifier.ts

```diff
@@ -549,14 +558,18 @@ export function createHoverifier<C extends object, D, A>({
               // It's important to do this before filtering otherwise navigating from
               // a position, to a line-only position, back to the first position would get ignored
               distinctUntilChanged((a, b) => isEqual(a, b)),
-              map(({ position, codeView, dom, ...rest }) => {
+              map(({ position, codeView, dom, overrideTokenize, ...rest }) => {
```

Comment:
Do you think, can we use a parameter default value assignment, like `overrideTokenize = tokenize, ...rest})` instead of `shouldTokenize` function?

Reply:
That's a great idea, but will we always use destructuring assignment? I think keeping it explicit will prevent regressions